docs/HowTo/kafka-cluster-migration/sync-connectors-from-cluster.md

```diff
@@ -23,4 +24,5 @@ them to a team in Klaw.
 
    ![image](../../../static/images/sync/SyncConnectorsFromCluster.png)
 
-In case a connector is deleted from the cluster but still exists in Klaw metadata, you can remove it by selecting **REMOVE FROM KLAW** at the end of the corresponding record.
+In case a connector is deleted from the cluster but still exists in Klaw metadata, you can remove it by selecting \*
```

Comment:
This end \* needs to be on the next line its not currently bolding "Remove from Klaw"

Reply:
Oh, good catch! That was a weird formatting issue, where the code formatting wants to do a line break and the **. 
I added `<!-- prettier-ignore -->` above that line, this will tell the formatter to not format the next line. 

https://github.com/Aiven-Open/klaw-docs/pull/87/commits/a01bc4f75ccb3a6efa5bd23342d3e64eb3231bf9

This should be an edge case, but I'll keep an eye out.